docker/intel-dgpu-driver.Dockerfile

```diff
@@ -34,6 +34,15 @@ RUN git clone -b ${I915_RELEASE} --single-branch https://github.com/intel-gpu/in
     && cp defconfigs/i915 .config \
     && make olddefconfig && make modules -j $(nproc) && make modules_install
 
+# Copy out-of-tree drivers to /opt/lib/modules/${KERNEL_FULL_VERSION}/
+RUN for file in $(find /lib/modules/${KERNEL_FULL_VERSION}/updates/ -name "*.ko"); do \
+    dname=$(dirname /opt$file); mkdir -p $dname; cp $file /opt$file; done
```

Comment:
Can you try command below? Make sure directory /opt already exists. 

```
RUN for file in $(find /lib/modules/${KERNEL_FULL_VERSION}/updates/ -name "*.ko"); do \
    cp $file /opt --parents; \
done
```

Reply:
This is new to me. :) Can you elaborate what I should try? Based on cp man page, --parents will use full source file name under DIRECTORY.

Reply:
Would you like me to try this?
```
RUN for file in $(find /lib/modules/${KERNEL_FULL_VERSION}/updates/ -name "*.ko"); do \
    dname=$(dirname /opt$file); mkdir -p $dname; cp $file /opt --parents; done
```

Reply:
Done!